lib/fedora_migrate/content_mover.rb

```diff
@@ -1,24 +1,44 @@
 module FedoraMigrate
   class ContentMover < Mover
 
+    include DatastreamVerification
+
+    class Report
+      attr_accessor :name, :mime_type, :original_date, :error
+      def success?
+        error.nil?
+      end
+    end
+
     def migrate
-      return nil_content_message if source.content.nil?
+      return report if nil_source
```

Comment:
@cam156 it's in mover. I made the decision that every mover should just return the report after it's done migrating. Reports are either simple classes or Struct objects. The reports get collected by the migrator as a big hash, and calling `.to_json` on them turns it into a big ball 'o JSON.